backend/LexData/LexBoxDbContext.cs

```diff
@@ -22,8 +25,14 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
     protected override void ConfigureConventions(ModelConfigurationBuilder builder)
     {
+        builder.Properties<List<FeatureFlag>>()
```

Comment:
this may not be needed, as EF core should support primitive arrays in the db natively.

Reply:
Primitive arrays, yes, but nowhere does the documentation mention enums, so I figured I needed to change `List<FeatureFlag>` into `List<string>` for EF Core to support it correctly. Once I have it working, I'll try removing this and see if it continues to work.

Reply:
Ahh, so Enums are primitives in dotnet because they are backed by a number type (int by default). So without a converter it will be an array of ints in the db, but we'd rather it be strings. So I think a converter is still required.